build.js

```diff
@@ -1,7 +1,7 @@
 // this is the build script
 // it takes files from source/ and transpiles them into build/
 // it takes components from components/ and substitutes them into all .html
-//  files in source that use the handlbar syntax of {{ ... }}
+//  files in source that use the handlebar syntax of {{ ... }}
 // all non-html files are simply copied
```

Comment:
can we add a bit more explanation of how the en and sk folders and this whole components thing works?

Reply:
Added some; let me know if more is needed.